internal/venice-common/src/main/java/com/linkedin/venice/helix/HelixCustomizedViewOfflinePushRepository.java

```diff
@@ -221,7 +221,12 @@ public void refreshRoutingDataForResource(String kafkaTopic) {
     throw new VeniceException("The function of refreshRoutingDataForResource is not implemented");
   }
 
-  private class StoreChangeListener implements StoreDataChangedListener {
+  // test only
+  public Map<String, Integer> getResourceToPartitionCountMap() {
```

Comment:
Can we make this package private if it's for tests only? Also, just to be extra cautious regarding misuse, it may be good to wrap it in `Collections:unmodifiableMap`.

Reply:
Sure, good idea to wrap it unmodifiable!